sig-contributor-experience/projects.md

```diff
@@ -0,0 +1,73 @@
+
+## Contributor Documentation, Workflow, and Automation
+Ensure the contribution process is well documented, discoverable, and consistent across repos to deliver the best contributor experience. Provide sufficient automation that direct write access to repos is no longer required.
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Contributor Guide](contributors/guide) | @castrojo | make contributor onboarding easier; first version | Q1
+Developer Guide | needs owner | a comprehensive guide for upstream developers to be a part of the Contributor Guide | Q2
+[New Community Website](https://github.com/kubernetes/community/issues/1819) | A new home for all things community -documentation, KEPs, Mentoring, + more | Q2 - ongoing
+[Label Documentation](https://gist.github.com/spiffxp/24937d8478853054c088ffc298021214) | @spiffxp | GitHub is rolling out a label description feature in the future; this is to document the label descriptions for contributors | Q1
+Issue Hygeine | @spzala | Produce clear understanding and documentation of issue hygeine | Q1
+PR Descriptions | @grodrigues, @spiffxp | Automation allowing people to edit pr descriptions for tldr or correct issue / pr; release notes based (design).  Define remaining work to be done. | Q1 - ongoing
+Cherry Pick Process | @spiffxp | Design and Prototype | Q2
+
+## Project-wide Communication Channels
+Build, curate, moderate, and make project wide communication channels accessible.
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+[Slack Admin Guidelines](communication/slack-guidelines.md) | @parispittman | first iteration of guidelines to administer slack for 30k+ users | Q1
+[Weekly K8s Community Meeting](events/community-meeting.md) | @castrojo, @parispittman | Smooth out weekly community meeting and make adjustments | Ongoing
+Roadshow | @parispittman | ContribEx to visit all SIG/WG meetings to deliver messages and collect feedback | Ongoing
+[Documentation](communication/README.md) | @parispittman, @castrojo | Create new communication directory and include all information about our communication channels | Q1, 2
+
+## DevStats
+https://k8s.devstats.cncf.io; work devstats into making it the one stop shop for all things upstream Kubernetes data. Looking for a Product Owner.
+
+Project | Owner(s)/Lead(s) | Description | Q1, Q2, Later
+---|---|---|---
+Graph of the Week | @castrojo | Present at least two graphs a month with explanation about why we care to the community meeting | Ongoing
+[User Guide](https://github.com/cncf/devstats/issues/35) | @jberkus, @parispittman, @tpepper | Create a v1 README for DevStats that explains each chart | Q2
+Data validation | @jberkus | Ensure that the data that running through the tool is accurate and aggregating the right information
```

Comment:
Ensure that the data that **is**  running

Reply:
Q1/Q2/Later column missing